Tests/XCUITests/FakespotTests.swift

```diff
@@ -0,0 +1,32 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+import XCTest
+
+class FakespotTests: BaseTestCase {
+    func testFakespotAvailable() {
```

Comment:
Please add the testrail link before the test

Reply:
If not created yet, we should create the tests there, I already talked to @abodea about this and he agrees on having the test suite there with this basic test for now

Reply:
I have created the test on TestRail.